### DISCUSSION TITLE
Parallel-Split Shadow Maps (PSSM) Maybe?

### DISCUSSION THREAD

**Discussion by Parallel-Paradox** [bevyengine/bevy#5802] [Ideas]
https://developer.nvidia.com/gpugems/gpugems3/part-ii-light-and-shadows/chapter-10-parallel-split-shadow-maps-programmable-gpus

Can this technique apply on bevy?

**Comment by superdump** [comment]
https://github.com/bevyengine/bevy/issues/3629
